### ISSUE TITLE
No such file or directory @ rb_sysopen

### ISSUE THREAD

**Issue by marksuurland** [chanks/que#209]
Just started to use que, but Im running into the following error: 'No such file or directory @ rb_sysopen - /User/.rvm/gems/ruby-2.2.5/gems/que-0.14.3/lib/que/migrations/4/up.sql'

When trying to run the migration file, like the example:

```
class CreateQueSchema < ActiveRecord::Migration[5.0]
	def up
    # Whenever you use Que in a migration, always specify the version you're
    # migrating to. If you're unsure what the current version is, check the
    # changelog.
    Que.migrate!(version: 4)
  end

  def down
    # Migrate to version 0 to remove Que entirely.
    Que.migrate!(version: 0)
  end
end

```
What im I doing wrong?

Thanks!

**Comment by nnc** [comment]
You seem to be using the 0.14.3 version of Que, where 3 is the newest migration version. Migration version 4 was added in Que 1.0, hence that files does not exists in your version of Que.

So you have two options:

1. Change the migration to target version 3 instead of 4.
2. Upgrade Que to version 1.0, and don't change your migration file.

**Comment by marksuurland** [comment]
Thanks for your quick reply. I guess updating to 1.0 (which is beta) is not the preferred solution since version 3 is the stable version right?

**Comment by nnc** [comment]
I haven't upgraded to 1.0 myself yet, but author of this gem is using 1.0 in production for months and it should be as stable the 0.x branch. So if I was just starting out with Que, I'd got for 1.0 now and avoid the upgrade later.

I *think* docs are already up to date with 1.0 as well.

**Comment by chanks** [comment]
Yeah, I merged the 1.0 branch into master, so I suppose you're looking at the 1.0 docs but using the 0.x branch. I'll add a notice to the README today to make that more clear.

I want to release an rc version soon, maybe that'll make people feel better about making the jump.

**Comment by marksuurland** [comment]
Like you guys mentioned, when upgrading to 1.0 this fixes my issue.
